src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -0,0 +1,484 @@
+                description: 'descLfVoltageInitMode',
+                values: {
+                    UNIFORM_VALUES: 'descLfUniformValues',
+                    PREVIOUS_VALUES: 'descLfPreviousValues',
+                    DC_VALUES: 'descLfDcValues',
+                },
+            },
+            noGeneratorReactiveLimits: {
+                type: TYPES.bool,
+                description: 'descLfNoGeneratorReactiveLimits',
+            },
+            twtSplitShuntAdmittance: {
+                type: TYPES.bool,
+                description: 'descLfTwtSplitShuntAdmittance',
+            },
+            readSlackBus: {
+                type: TYPES.bool,
+                description: 'descLfReadSlackBus',
+            },
+            writeSlackBus: {
+                type: TYPES.bool,
+                description: 'descLfWriteSlackBus',
+            },
+            distributedSlack: {
+                type: TYPES.bool,
+                description: 'descLfDistributedSlack',
+            },
+            shuntCompensatorVoltageControlOn: {
+                type: TYPES.bool,
+                description: 'descLfShuntCompensatorVoltageControlOn',
+            },
+            dcUseTransformerRatio: {
+                type: TYPES.bool,
+                description: 'descLfDcUseTransformerRatio',
+            },
+        };
+
+        return (
+            lfParams && (
+                <>
+                    {MakeAdvancedParameterButton(
+                        showAdvancedLfParams,
+                        'showAdvancedParameters',
+                        () => setShowAdvancedLfParams(!showAdvancedLfParams)
+                    )}
+                    {showAdvancedLfParams &&
+                        makeComponentsFor(
+                            defParams,
+                            lfParams,
+                            commitLFParameter
+                        )}
+                </>
+            )
+        );
+    };
+
+    function MakeAdvancedParameterButton(showOpenIcon, label, callback) {
+        return (
+            <>
+                <Grid item xs={12} className={classes.advancedParameterButton}>
+                    <Button
+                        startIcon={<SettingsIcon />}
+                        endIcon={
+                            showOpenIcon ? (
+                                <CheckIcon style={{ color: 'green' }} />
+                            ) : undefined
+                        }
+                        onClick={callback}
+                    >
+                        <FormattedMessage id={label} />
+                    </Button>
+                </Grid>
+            </>
+        );
+    }
+
+    function makeComponentsFor(defParams, params, setter) {
+        return Object.keys(defParams).map((key) => (
+            <Grid container spacing={1} paddingTop={1} key={key}>
+                {makeComponentFor(defParams[key], key, params, setter)}
+                <Grid item xs={12}>
```

Comment:
You can keep the function LineSeparator you defined in line-separator but put it in parameters.js, not in a new file with only this method.

Reply:
Actually, put this function in the dialogUtils.js file. It's the file for such components.
You can also put the slider in this file.